src/plugins/intel_gpu/tests/test_utils/float16.h

```diff
@@ -88,6 +92,8 @@ inline bool operator<(const FLOAT16 &v1, const float &v2) { return (float)v1 < v
 
 inline bool operator==(const FLOAT16 &v1, const FLOAT16 &v2) { return v1.v == v2.v; }
 
+inline bool operator==(const FLOAT16 &v1, const int &v2) { return v1.v == v2; }
+
```

Comment:
where are these changed used?

Reply:
This changed were needed for ngraph::runtime::reference::scatterNdUpdate<FLOAT16, FLOAT16>. The overload for the operators were ambiguous, causing an error, so I added the changes.